optimize/backend/src/main/java/io/camunda/optimize/service/db/repository/es/ProcessRepositoryES.java

```diff
@@ -59,7 +59,8 @@ public Map<String, ProcessOverviewDto> getProcessOverviewsByKey(
         OptimizeSearchRequestBuilderES.of(
             b ->
                 b.optimizeIndex(esClient, PROCESS_OVERVIEW_INDEX_NAME)
-                    .query(q -> q.ids(i -> i.values(processDefinitionKeys.stream().toList()))));
+                    .query(q -> q.ids(i -> i.values(processDefinitionKeys.stream().toList())))
```

Comment:
According to the ticket, the issue is also present with OpenSearch, but the fix is only for ElasticSearch?

Reply:
No, for OpenSearch both code and UI behaiour look good.